extension/lib/CommonStorage.jsm

```diff
@@ -0,0 +1,44 @@
+ /* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const { utils: Cu } = Components;
+
+Cu.import("resource://gre/modules/XPCOMUtils.jsm");
+
+XPCOMUtils.defineLazyModuleGetter(
+  this, "NewsIndexedDB", "resource://pioneer-study-online-news/lib/NewsIndexedDB.jsm"
+);
+
+this.EXPORTED_SYMBOLS = ["CommonStorage"];
+
+
+this.CommonStorage = {
+  getStore() {
+    return NewsIndexedDB.getStore("common");
```

Comment:
"common" seems like a bad choice of name here. Maybe something more descriptive?